.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+  # Enable version updates for git submodules
+  - package-ecosystem: "gitsubmodule"
+    # Look for `.gitmodules` in the `root` directory
+    directory:         "/"
+    # Check the submodules for updates every week
+    schedule:
+      interval: "weekly"
+    assignees:
+      - "burgholzer"
+    reviewers:
+      - "burgholzer"
```

Comment:
you can remove the `reviewers` part in all these configurations. you already get notified if you are assigned to a PR so this should not be necessary.

Reply:
Thanks